client/client.go

```diff
@@ -266,6 +289,7 @@ func MakeConfiguration(file string) (err error) {
 	}
 	fmt.Fprintf(fd, "[api]\n\turl = \"%s\"\n", cfg.API.URL)
 	fmt.Fprintf(fd, "[gpg]\n\thome = \"%s\"\n\tkeyid = \"%s\"\n", cfg.GPG.Home, cfg.GPG.KeyID)
+	fmt.Fprintf(fd, "[TerminalConf]\n\tcolor = off\n")
```

Comment:
It seems like a lengthy way. I am thinking of another function which parse the text which is going to print and remove all the escape sequence from it if color flag is off. 
````
func checkColor(s string) string {
	if color == on {
		return s
	} else {
		s = strings.Replace(s, "\x1b[0m", "", -1)
		var buffer bytes.Buffer
		color_code_flag := 0
		for i := 0; i < len(s); i++ {
			if i+4 <= len(s) && string(s[i]) == "[" && (string(s[i+3]) == ";" || string(s[i+4]) == ";") {
				color_code_flag = 1
			} else if string(s[i]) == "m" {
				color_code_flag = 0
			}
			if color_code_flag == 0 {
				buffer.WriteString(string(s[i]))
			}
		}
		return buffer.String()
	}

}
```
which is call as 
	`fmt.Println(checkColor("\x1b[31;1m+------\x1b[0m"))`
@ameihm0912 please give feedback and apologies for taking time.